xlsx/src/import/worksheets.rs

```diff
@@ -146,20 +146,23 @@ fn load_columns(ws: Node) -> Result<Vec<Col>, XlsxError> {
     Ok(cols)
 }
 
-fn load_merge_cells(ws: Node) -> Result<Vec<String>, XlsxError> {
+fn load_merge_cells_nodes(ws: Node) -> Result<Vec<MergeCell>, XlsxError> {
     // 18.3.1.55 Merge Cells
     // <mergeCells count="1">
     //    <mergeCell ref="K7:L10"/>
     // </mergeCells>
-    let mut merge_cells = Vec::new();
+    let mut merge_cells: Vec<MergeCell> = Vec::new();
     let merge_cells_nodes = ws
         .children()
         .filter(|n| n.has_tag_name("mergeCells"))
         .collect::<Vec<Node>>();
     if merge_cells_nodes.len() == 1 {
         for merge_cell in merge_cells_nodes[0].children() {
             let reference = get_attribute(&merge_cell, "ref")?.to_string();
-            merge_cells.push(reference);
+            let merge_cell_st = reference.split(":").nth(0).unwrap();
```

Comment:
We should never use unwrap in production code. And reference needs proper parsing. There is a function `parse_range` in this file you can use

Reply:
Understood, will be using this